ansible/configs/studentvm/default_vars_ec2.yml

```diff
@@ -0,0 +1,50 @@
+### AWS EC2 Environment settings
+
+# The region to be used, if not specified by -e in the command line
+aws_region: us-east-1
+
+# The key that is used to
+key_name: "default_key_name"
+
+## Networking (AWS)
+
+### Route 53 Zone ID (AWS)
+# This is the Route53 HostedZoneId where you will create your Public DNS entries
+#
+# HostedZoneId needs to come from the account that is being used. It also needs to match
+# subdomain_base_suffix
+HostedZoneId: Z3IHLWJZOU9SRT
+
+subdomain_base_short: "{{ guid }}"
+subdomain_base_suffix: ".example.opentlc.com"
+subdomain_base: "{{subdomain_base_short}}{{subdomain_base_suffix}}"
+
+## Environment Sizing
+
+studentvm_instance_type: "t3a.large"
+studentvm_instance_image: RHEL81GOLD
+studentvm_rootfs_size: 200
```

Comment:
I don't actually know what the cost is here, but considering how many clientvm type workloads we configure, this can probably be defaulted to much smaller. If it needs to be large for some reason, it can be overidden in agnosticv

Reply:
Yeah. t3a.medium should be fine.

Reply:
i meant for disk sizes...not instance sizes. Although both, I guess.